### DISCUSSION TITLE
Update `fill` and responsive examples to use correct `sizes` prop

### DISCUSSION THREAD

**Discussion by duveborg** [vercel/next.js#64644] [Ideas]
### What version of Next.js are you using?

11

### What version of Node.js are you using?

12

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying your application?

aws

### Describe the Bug

In the gallery example, a next image with a parent element that has a set size, it still loads an image the size of the viewport

can be seen in this example https://image-component.nextjs.gallery/layout-fill

![image](https://user-images.githubusercontent.com/840115/139504173-46cbaca3-7cfd-4c74-9030-c6df509fd750.png)

### Expected Behavior

Should load smaller images

### To Reproduce

check official example https://image-component.nextjs.gallery/layout-fill

**Comment by styfle** [comment]
This isn't a bug, this is working as expected.

If you don't expect your image to take up the full width of the viewport, you can set the [`sizes`](https://nextjs.org/docs/api-reference/next/image#sizes) prop.

That being said, the example should probably be updated to demonstrate that usage 👍

**Comment by lachlanjc** [comment]
Can close since the new Image component doesn't have `layout` anymore ☺️

**Comment by styfle** [comment]
We still have examples for the legacy component. And also the new component should have `sizes` in case of responsive. So the issue is still relevant.
